.evergreen/utils.sh

```diff
@@ -9,11 +9,11 @@ set -o xtrace
 createvirtualenv () {
     PYTHON=$1
     VENVPATH=$2
-    if $PYTHON -m venv -h>/dev/null; then
+    if $PYTHON -m virtualenv --version; then
```

Comment:
Workaround until we get the right Ubuntu package installed to actually make venv work for the system python.

Reply:
Ack. Does this work on mac? I remember something about virtualenv not working there which  is one reason we switched to venv.

Reply:
It appears to work: https://evergreen.mongodb.com/task/mongo_python_driver_test_macos_encryption__platform~macos_1014_auth~auth_ssl~nossl_encryption~encryption_test_4.2_standalone_patch_1d7b9a80b91315f1af4b88577e20ab4ac9e55e8f_61b3ea0e306615750a4d77c2_21_12_11_00_00_34